ynr/apps/twitterbot/management/commands/twitterbot_update_usernames.py

```diff
@@ -117,12 +117,25 @@ def handle_person(self, person):
                         ]
                     )
                 )
-                person.identifiers.create(
-                    scheme="twitter",
-                    identifier=self.twitter_data.screen_name_to_user_id[
-                        screen_name.lower()
-                    ],
+
+                person.tmp_person_identifiers.update_or_create(
+                    person=person,
+                    value_type="twitter_username",
+                    value=screen_name.lower(),
+                    defaults={
+                        "internal_identifier": self.twitter_data.screen_name_to_user_id[
+                            screen_name.lower()
+                        ]
+                    },
                 )
+                """
+                person.identifiers.create(
+-                    scheme="twitter",
+-                    identifier=self.twitter_data.screen_name_to_user_id[
+-                        screen_name.lower()
+-                    ],
+-                )"""
```

Comment:
This block looks like its not supposed to be here?

Reply:
Whoops, nope!